package.json

```diff
@@ -5,6 +5,9 @@
         "curator"
     ],
     "scripts": {
-        "build" : "npm run build --workspaces"
+        "build": "npm run build --workspaces"
+    },
+    "dependencies": {
+        "sass": "^1.82.0"
```

Comment:
**question:** Do you really need sass here ? You only use variables that can be easily replaced by CSS variables ?